content/blog/2018/migrating-gateway/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Migrating to the Istio Gateway
+subtitle: Replace your Ingresses with Gateways using Istio 1.0
+description: Converting and replacing your Kubernetes Ingresses with Istio Gateways
+publishdate: 2018-08-15
+attribution: The Istio Team
```

Comment:
would not this be you?  I think the attribution is istio team for release announcement.